pages/contributors.js

```diff
@@ -0,0 +1,113 @@
+import { React, useState, useEffect} from 'react'
+import Page from '../components/Page'
+
+export default function About() {
+    const [contributors, setContributors] = useState([])
+    
+    useEffect(() => {
+      const fetchContributors = async () => {
+        const response = await fetch(
+          'https://api.github.com/repos/carbon-app/carbon/contributors'
+        )
+        const contributors = await response.json()
+  
+        setContributors(
+          contributors.filter((contributor) => !contributor.login.includes('bot'))
```

Comment:
I think you will want to filter by logins ending in `[bot]` — right now this will even filter legitimate users, like someone names `bottom`, for example 🙂

Reply:
Makes sense!